examples/ase/ase_openmm_neuraminidase.ipynb

```diff
@@ -342,59 +413,58 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we've got an ASE Atoms object, and a calculator, we can set up NanoVer with ASE as [usual](./ase_basic_example.ipynb).\n",
+    "Now we've got an ASE Atoms object, and a calculator, we can set up NanoVer with ASE as [usual](./ase_basic_example.ipynb). \n",
     "The only difference here is that we swap out the default way of sending frames with a specially made one, `openmm_ase_frame_adaptor`, for OpenMM that knows about OpenMM topology"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 15,
-   "metadata": {},
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-10-11T11:03:13.895040Z",
+     "start_time": "2024-10-11T11:03:13.889625Z"
+    }
+   },
    "outputs": [],
    "source": [
     "from ase.md import Langevin\n",
     "import ase.units as ase_units\n",
-    "dynamics = Langevin(atoms, timestep=1.0 * ase_units.fs, temperature_K=300 * ase_units.kB, friction=1.0e-03)"
+    "dynamics = Langevin(atoms, timestep=1.0 * ase_units.fs, temperature_K=300, friction=1.0e-03)"
```

Comment:
When setting the temperature using `temperature_K` I think you only need to specify the number in Kelvin, rather than converting to units of kT by multiplying by the Boltzmann constant kB (see the first note on [this page](https://wiki.fysik.dtu.dk/ase/ase/md.html)